wildfly-mixin/config.libsonnet

```
@@ -1,17 +1,33 @@
 {
-  _config+:: {
-    enableMultiCluster: false,
-    multiclusterSelector: 'job=~"$job"',
-    wildflySelector: if self.enableMultiCluster then 'job=~"$job", cluster=~"$cluster"' else 'job=~"$job"',
-    dashboardTags: ['wildfly-mixin'],
-    dashboardPeriod: 'now-1h',
-    dashboardTimezone: 'default',
-    dashboardRefresh: '1m',
+  local this = self,
+  filteringSelector: 'job="integrations/wildfly"',
+  groupLabels: ['job', 'cluster'],
+  logLabels: ['job', 'cluster', 'instance'],
+  instanceLabels: ['instance'],
 
-    // alerts thresholds
-    alertsErrorRequestErrorRate: '30',
-    alertsErrorRejectedSessions: '20',
+  uid: 'wildfly',
+  dashboardTags: [self.uid],
```

```suggestion
  dashboardTags: [self.uid+-'mixin'],
```

This'd be a breaking default otherwise